crates/flux-fhir-analysis/src/lib.rs

```diff
@@ -446,8 +446,12 @@ fn variants_of(
     Ok(variants)
 }
 
-fn fn_sig(genv: GlobalEnv, def_id: LocalDefId) -> QueryResult<rty::EarlyBinder<rty::PolyFnSig>> {
-    let def_id = genv.maybe_extern_id(def_id);
+fn fn_sig(
+    genv: GlobalEnv,
+    orig_def_id: LocalDefId,
```

Comment:
I like shadowing the input so we don't use it by mistake.

Reply:
Yes, I reverted.